esmvaltool_sample_data/sample_data.py

```diff
@@ -112,6 +109,26 @@ def select_host(hosts, preferred_hosts, ignore_hosts):
     return hosts[0]
 
 
+def select_latest_versions(datasets: dict) -> dict:
+    """Return a dict with only the latest version of each dataset.
+
+    If `datasets` contains multiple versions of a dataset, return only
```

Comment:
Could you add [a numpy style docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html), so this function is documented in the same way as the other functions in this file?